pubspec.yaml

```diff
@@ -4,13 +4,14 @@ version: 0.7.0
 homepage: https://github.com/BrunoJurkovic/flip_card/
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.17.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
 dev_dependencies:
+  flutter_lints: ^2.0.1
```

Comment:
I personally prefer the [lint](https://pub.dev/packages/lint) package as it's pretty strict. That being said, any linter is a big improvement already :D

Reply:
No it's ok, leave it as is.